plasmapy/formulary/frequencies.py

```diff
@@ -516,3 +519,101 @@ def upper_hybrid_frequency(B: u.T, n_e: u.m**-3) -> u.rad / u.s:
+
+
+@validate_quantities(
+    validations_on_return={
+        "units": [u.rad / u.s, u.Hz],
+        "equivalencies": [(u.cy / u.s, u.Hz)],
+    }
+)
+@angular_freq_to_hz
+def Buchsbaum_frequency(
+    B: u.T,
+    n1: u.m**-3,
+    n2: u.m**-3,
+    ion1: ParticleLike,
+    ion2: ParticleLike,
+    Z1: Optional[float] = None,
+    Z2: Optional[float] = None
+) -> u.rad / u.s:
+    r"""
+    Return the Buchsbaum frequency for a two-ion-species plasma.
+
+    Parameters
+    ----------
+    B : `~astropy.units.Quantity`
+        The magnetic field magnitude in units convertible to tesla.
+    n1 : `~astropy.units.Quantity`
+        Particle number density of ion species #1 in units convertible to m\ :sup:`-3`.
+    n2 : `~astropy.units.Quantity`
+        Particle number density of ion species #2 in units convertible to m\ :sup:`-3`.
+    ion1 : `~plasmapy.particles.particle_class.Particle`
+        Representation of ion species #1 (e.g., 'p' for protons, 'D+'
+        for deuterium, or 'He-4 +1' for singly ionized helium-4). If no
+        charge state information is provided, then species #1 is assumed
+        to be singly charged.
+    ion2 : `~plasmapy.particles.particle_class.Particle`
+        Representation of ion species #2 (same behavior as for ion1).
+    Z1 : `float` or `~astropy.units.Quantity`, optional
+        The charge state for ion species #1. If not provided, it defaults to the charge number of ``ion1``.
+    Z2 : `float` or `~astropy.units.Quantity`, optional
+        The charge state for ion species #2. If not provided, it defaults to the charge number of ``ion2``.
+
+    Returns
+    -------
+    omega_BB : `~astropy.units.Quantity`
+        The Buchsbaum frequency of the plasma in units of radians per second.
+        Setting keyword ``to_hz=True`` will apply the factor of :math:`1/2π`
+        and yield a value in Hz.
+
+    Raises
+    ------
+    `TypeError`
+        If the magnetic field is not a `~astropy.units.Quantity` or
+        ``particle`` is not of an appropriate type.
+    `ValueError`
+        If the magnetic field contains invalid values or particle cannot
+        be used to identify a particle or isotope.
+
+    Warns
+    -----
+    : `~astropy.units.UnitsWarning`
+        If units are not provided, SI units are assumed.
+
+    Notes
+    -----
+    In a magnetized plasma, the presence of two ion species allows the
+    perpendicular component of the cold-plasma dielectric coefficient
+    :math:`\epsilon_{\perp}` to vanish at an angular frequency referred
+    to as the Buchsbaum frequency :cite:p:`buchsbaum:1960`, also called
+    the bi-ion hybrid resonance frequency :cite:p: `thompson:1995`, or ion-ion hybrid frequency
+    :cite:p:`vincena:2013`.  This frequency can be defined as:
+
+    .. math::
+        \omega_{BB} \equiv \sqrt{\frac{\omega_{p1}^{2}\omega_{c2}^{2}
+            + \omega_{p2}^{2}\omega_{c1}^{2}}{\omega_{p2}^{2}+\omega_{p2}^{2}}}
```

Comment:
```suggestion
        ω_{BB} ≡ \sqrt{\frac{ω_{p1}^2 ω_{c2}^2
            + ω_{p2}^2 ω_{c1}^2}{ω_{p2}^2 + ω_{p2}^2}}
```
Might make the math a bit more readable when using `help()`.

Reply:
Again, I don't speak UTF, so this is foreign to me, but so let it be written, so let it be done!